appveyor.yml

```diff
@@ -1,5 +1,5 @@
 version: '{build}'
-os: Visual Studio 2017
+os: Visual Studio 2019 Preview
 install:
     - set PATH=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\;%PATH%
```

Comment:
You may have to change this path too, I don't use appveyor, but I don't think that the "os" has VS2017 build with a full VS2019